presto-docs/src/main/sphinx/index.rst

```diff
@@ -3,19 +3,28 @@ Presto Documentation
 ####################
 
 .. toctree::
+   :hidden:
+
+   develop
```

Comment:
Maybe we have. If that's the case, then there's no point in me carrying developer's guide changes for our docs :)